### PR TITLE
Raise exception if the timestamp_field is specified but not found in the schema

### PR DESCRIPTION
**Pull request by jiangxin369**
## What is the purpose of the change

Check if the schema contains timestamp field and key fields if they are set.

## Brief change log

  - Check if the schema contains timestamp field and key fields if they are set.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (no)
  - The public API: (no)

## Documentation

  - Does this pull request introduce a new feature? (no)
  - If yes, how is the feature documented? (not applicable)

### CODE REVIEW

**Comment by jiangxin369** [comment]
@Sxnan Could you help review this PR?

File: python/feathub/feature_tables/feature_table.py [hunk 1]
```diff
@@ -74,6 +74,20 @@ def __init__(
         self.properties = properties
         self.schema = schema
 
+        if schema:
```
Comment (Sxnan) [inline python/feathub/feature_tables/feature_table.py:77]: Is it safer if we explicitly check `if schema is not None`? Same for the other None check.

**Review by Sxnan — COMMENTED** [review]
Thanks for the PR! Just left one minor comment below.

**Comment by Sxnan** [comment]
Thanks for your update! LGTM. @lindong28 Could you help take a look at this PR?

**Comment by lindong28** [comment]
Thanks for the PR. LGTM.

**Pull request merged**